jscomp/ml/types.ml

```diff
@@ -303,7 +303,7 @@ type label_description =
     lbl_arg: type_expr;                 (* Type of the argument *)
     lbl_mut: mutable_flag;              (* Is this a mutable field? *)
     lbl_pos: int;                       (* Position in block *)
-    lbl_all: label_description array;   (* All the labels in this type *)
+    mutable lbl_all: label_description array;   (* All the labels in this type *)
```

Comment:
add comment here: this should really *not* be mutated, only used selectively by the compiler for a specific reason